lib/workers/repository/updates/flatten.ts

```diff
@@ -51,10 +51,12 @@ export async function flattenUpdates(
               updateConfig.packageRules
             );
             // apply major/minor/patch/pin/digest
-            updateConfig = mergeChildConfig(
-              updateConfig,
-              updateConfig[updateConfig.updateType]
-            );
+            updateConfig.updateTypes?.forEach((updateType) => {
+              updateConfig = mergeChildConfig(
+                updateConfig,
+                updateConfig[updateType]
+              );
+            });
```

Comment:
Another one potentially dangerous place, though it's working for now (with single values)